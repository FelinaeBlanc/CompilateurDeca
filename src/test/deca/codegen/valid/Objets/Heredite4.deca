class A {
   int getVal(){
        return 1;
   }
}

class B extends A {

   int getVal(){
        return 2;
   }

}

{
  A a = new A();
  A b = new B();
  println("a.getVal() = ", a.getVal());
  println("b.getVal() = ", b.getVal());
}
