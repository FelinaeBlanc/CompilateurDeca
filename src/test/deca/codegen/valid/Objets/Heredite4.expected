a.getVal() = 1
b.getVal() = 2
